src/commands/synthetics/run-test.ts

```diff
@@ -220,11 +253,18 @@ export class RunTestCommand extends Command {
   }
 }
 
+const removeUndefinedValues = <T extends {[key: string]: any}>(object: T): T => {
+  const newObject = {...object}
+  Object.keys(newObject).forEach((k) => newObject[k] === undefined && delete newObject[k])
+
+  return newObject
+}
+
 RunTestCommand.addPath('synthetics', 'run-tests')
 RunTestCommand.addOption('apiKey', Command.String('--apiKey'))
 RunTestCommand.addOption('appKey', Command.String('--appKey'))
 RunTestCommand.addOption('configPath', Command.String('--config'))
+RunTestCommand.addOption('fileGlobs', Command.Array('-f,--files'))
 RunTestCommand.addOption('publicIds', Command.Array('-p,--public-id'))
-RunTestCommand.addOption('testSearchQuery', Command.String('-s,--search'))
 RunTestCommand.addOption('shouldOpenTunnel', Command.Boolean('-t,--tunnel'))
```

Comment:
This name doesn't need to be as explicit anymore IMO.
Should be unified as `tunnel`.